tabbycat/breakqual/tests/test_liveness.py

```diff
@@ -61,60 +61,92 @@ def test_case_australs_2015_open(self):
+        self.assertGreaterEqual(safe, 11)  # All teams on 13 will break (worst case)
+        self.assertLessEqual(dead, 3)  # Some teams on 10 will break (best case)
 
     def test_case_melbourne_mini_2017(self):
         safe, dead = liveness_bp(True, 6, 8, 20, 6, [])
-        self.assertGreaterEqual(safe, 11) # All teams on 13 will break (worst case)
-        self.assertLessEqual(dead, 6) # Some teams on 10 will break (best case)
+        self.assertGreaterEqual(safe, 11)  # All teams on 13 will break (worst case)
+        self.assertLessEqual(dead, 6)  # Some teams on 10 will break (best case)
 
     def test_case_usu_iv_2016_penultimate(self):
         safe, dead = liveness_bp(True, 4, 8, 30, 5, [])
-        self.assertGreaterEqual(safe, 10) # All teams on 11 will break (worst case)
-        self.assertLessEqual(dead, 2) # Some teams on 9 will break (best case)
+        self.assertGreaterEqual(safe, 10)  # All teams on 11 will break (worst case)
+        self.assertLessEqual(dead, 2)  # Some teams on 9 will break (best case)
 
     def test_case_usu_iv_2016(self):
         safe, dead = liveness_bp(True, 5, 8, 30, 5, [])
-        self.assertGreaterEqual(safe, 10) # All teams on 11 will break (worst case)
-        self.assertLessEqual(dead, 5) # Some teams on 9 will break (best case)
+        self.assertGreaterEqual(safe, 10)  # All teams on 11 will break (worst case)
+        self.assertLessEqual(dead, 5)  # Some teams on 9 will break (best case)
 
     def test_case_abp_2017_efl(self):
         safe, dead = liveness_bp(False, 6, 8, 21, 6, [10, 9, 9, 8, 7, 7, 7, 6, 6])
-        self.assertGreaterEqual(safe, 8) # All teams on 8 broke
-        self.assertLessEqual(dead, 3) # Some teams on 7 broke
+        self.assertGreaterEqual(safe, 8)  # All teams on 8 broke
+        self.assertLessEqual(dead, 3)  # Some teams on 7 broke
 
     def test_bp_coefficients(self):
         data = {
             0: [1],
             1: [1, 1, 1, 1],
             2: [1, 2, 3, 4, 3, 2, 1],
-            10: [1, 10, 55, 220, 705, 1902, 4455, 9240, 17205, 29050, 44803, 63460, 82885, 100110, 112035, 116304, 112035, 100110, 82885, 63460, 44803, 29050, 17205, 9240, 4455, 1902, 705, 220, 55, 10, 1],  # noqa: E501
+            10: [
+                1,
+                10,
+                55,
+                220,
+                705,
+                1902,
+                4455,
+                9240,
+                17205,
+                29050,
+                44803,
+                63460,
+                82885,
+                100110,
+                112035,
+                116304,
+                112035,
+                100110,
+                82885,
+                63460,
+                44803,
+                29050,
+                17205,
+                9240,
+                4455,
+                1902,
+                705,
+                220,
+                55,
+                10,
+                1,
+            ],  # noqa: E501
```

Comment:
Changes like this seem like clearly a bad idea to me. This had a `# noqa` ignore command for flake8, how do we do the same with black?